examples/Handpose-p5-start-stop/sketch.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let handpose;
+let video;
+let hands = [];
+let isDetecting = false;
+
+function preload() {
+  // Load the handpose model.
+  handpose = ml5.handpose();
+  console.log("preload");
+}
+
+function setup() {
+  createCanvas(640, 480);
+  fill(0, 255, 0);
+  textSize(25);
+  noStroke();
```

Comment:
I think it's generally more helpful and clear to place these where the drawing is happening (even though it's redundant). With these in `setup()`  I find it can cause confusion for students adapting the examples to include more elaborate drawing.

Reply:
I have moved these to their corresponding locations!